src/core/execute-serialize.c

```diff
@@ -2005,7 +2005,10 @@ static int exec_context_serialize(const ExecContext *c, FILE *f) {
                 if (!key)
                         return log_oom_debug();
 
-                if (asprintf(&value, "%04o", c->directories[dt].mode) < 0)
+                if (asprintf(&value, "%04o %" PRIu64 " %" PRIu32 " %d", c->directories[dt].mode,
```

Comment:
do not change serialization of already existing fields. we need to retain compat, serialization/deserialization is used for upgrades after all